pypeit/wavecalib.py

```diff
@@ -548,19 +574,19 @@ def build_wv_calib(self, arccen, method, skip_QA=False):
                                              nsnippet=self.par['nsnippet'])
                                              #debug=True, debug_reid=True, debug_xcorr=True)
         elif self.par['method'] == 'echelle':
```

Comment:
Can you add a TODO somewhere about merging this eventually with "reidentify" for fixed format echelles?

Reply:
added